Add ASETrajectory

yqshao opened the pull request:
Hi, I tried to add the ability to show ASE trajectory in nglview.
I think ase_traj acts just like a list of ase_atoms.
I didn't put any .traj files, but you can generate one with this.
```python
# Generate a simple ase traj
from ase import Atoms, Atom
from ase.io.trajectory import Trajectory
co = Atoms([Atom('C', (0, 0, 0)),
            Atom('O', (0, 0, 1.2))])
traj = Trajectory('1.traj', 'w', co)
for i in range(5):
    co.positions[1, 2] += 0.1
    traj.write()
# Test viewing
t=Trajectory('1.traj')
view=nv.show_asetraj(t)
view.add_ball_and_stick()
view
```

hainm commented:
hi, can you post a screenshot of your notebook here? (for future reference). thx.

hainm commented on nglview/adaptor.py:
```diff
@@ -348,4 +349,37 @@ def get_structure_string(self):
+        return pdb_string
+
+
+@register_backend('asetraj')
```
`asetraj` should be `ase` since this code will check the package name.

You can test that by

```
from ase.io.trajectory import Trajectory
traj = Trajectory('example.traj')
view = nglview.NGLWidget()
view.add_trajectory(traj) 
# so you don't need to create ase_traj = ASETrajectory(traj)
view
```

hainm commented:
Aslo, please update README.md file too. https://github.com/arose/nglview#usage

yqshao commented:
Here is the screenshot, 

![2017-02-23-212755_926x484_scrot](https://cloud.githubusercontent.com/assets/22888066/23260901/0440093e-fa0f-11e6-9171-ae92849a1440.png)

I fixed the backbend issue, and added a entry in the function table.
Also converted a traj file from the md_1u19 file from datafiles to make the examples work.

hainm commented:
Thank you.

hainm commented:
@yqshao ah, can you double check why there is no video player displayed? If `n_frames` > 1, there should be.
